static/app/types/organization.tsx

```diff
@@ -172,6 +172,15 @@ export interface TeamMember extends Member {
   teamSlug?: string;
 }
 
+/**
+ * Missing org members detected through commit authors
+ */
+export interface MissingMember {
+  commitCount: number;
+  email: string;
+  userId: string;
```

Comment:
Might be good to leave a comment on the purpose of "userId".

Reply:
oh actually the API response does return `externalId`